plextraktsync/trakt/TraktUserList.py

```diff
@@ -101,6 +101,15 @@ def add(self, m: Media):
 
         self.logger.info(f'Adding {m.title_link} ({m.plex_key}) to Plex list {self.title_link}', extra={"markup": True})
 
+        # Report duplicates
+        duplicates = [p for _, p in self.plex_items if p.key != m.plex_key and p == m.plex]
+        for p in duplicates:
+            msg = f'Duplicate {p.title_link} ({p.key}) with {m.title_link} {m.plex_key}'
```

Comment:
I'd remove parentheses around first plex key. Too mush parentheses with years make it unreadable.

Reply:
how about

```
WARNING  Duplicate Gone with the Wind (1939) #14257 with Gone with the Wind (1940) #15133
```